package.json

```diff
@@ -11,7 +11,7 @@
     "lint-css": "stylelint 'public/*.css' '.storybook/*.css' 'src/**/*.svelte' 'stories/**/*.svelte'",
     "lint-js": "eslint . .storybook",
     "skip-taskcluster": "node -e \"process.exit(!('TASK_ID' in process.env))\"",
-    "start": "sirv public",
+    "start": "sirv public --quiet",
```

Comment:
FWIW I don't think this is a super useful target anymore, since the Svelte app only works in webextension contexts. @Dexterp37 I think it might be more useful to have `npm run start` run the webextension w/o watch, wdyt?

Reply:
I think we can remove this for now and then add back "start" if we think it's useful. But no strong opinion!